packages/destination-actions/src/destinations/tiktok-audiences/api/index.ts

```diff
@@ -0,0 +1,121 @@
+import type { RequestClient, ModifiedResponse } from '@segment/actions-core'
+import type { Payload } from '../addUser/generated-types'
+import { BASE_URL, TIKTOK_API_VERSION } from '../constants'
+import type { GetAudienceAPIResponse, CreateAudienceAPIResponse } from '../types'
+import { DynamicFieldResponse } from '@segment/actions-core'
+
+interface AdvertiserInfoItem {
+  advertiser_id: string
+  name: string
+}
```

Comment:
Add extra space after interface declaration:

```suggestion
}

```